src/js/controller/settings/exportimage/ImageExportController.js

```diff
@@ -17,6 +17,11 @@
     this.addEventListener(scalingFactorInput, 'input', this.onScalingFactorChange_);
     this.updateScalingFactorText_(scalingFactorInput.value);
 
+    var framesPerRowsInput = document.querySelector('.export-frames-per-rows-field');
+    framesPerRowsInput.value = pskl.UserSettings.get(pskl.UserSettings.FRAMES_PER_ROWS);
+    this.addEventListener(framesPerRowsInput, 'change', this.onFramesPerRowsChange_);
+    this.addEventListener(framesPerRowsInput, 'input', this.onFramesPerRowsChange_);
+
```

Comment:
I put it just next to the "scaling fator" controlling to follow the same logic. Should I move both on the export controller directly ?